### ISSUE TITLE
Is anyone interested in company scraper?

### ISSUE THREAD

**Issue by ishandutta2007** [linkedtales/scrapedin#115]
I don't see much progress being made [here](https://github.com/linkedtales/scrapedin/tree/master/src/company)
I have initiated some work on my fork, if anyone else is interested in scraping companies please collab to get it done quickly.

**Comment by ganeshBUnbound** [comment]
Yeah @ishandutta2007 , we would love to. Let me know how and what thing you have in mind

> I don't see much progress being made [here](https://github.com/linkedtales/scrapedin/tree/master/src/company)
> I have initiated some work on my fork, if anyone else is interested in scraping companies please collab to get it done quickly.

**Comment by ishandutta2007** [comment]
> Yeah @ishandutta2007 , we would love to. Let me know how and what thing you have in mind
> 
> >

Please check my fork. Its been long time, I don't really remember what I added to it. Idea is nothing out of the world, it's to do the same template based scraping for companies as well just like how we are doing for profile pages.
Clone my Fork and try to build on top of it and send merge requests to me. 
I will in turn send the combined merge to the parent repo.
